web/modules/custom/ct_reports/ct_reports.info.yml

```
@@ -0,0 +1,4 @@
+name: Contribution Tracker Reports
+type: module
+description: Provides functionality to generate reports based on contribution data.
+core_version_requirement: ^8.9 || ^9
```

```suggestion
core_version_requirement: ^9
```

Let's skip supporting Drupal 8.9.